posthog/queries/funnels/funnel_event_query.py

```diff
@@ -111,7 +111,7 @@ def get_query(
             SELECT {all_fields}
             FROM events {self.EVENT_TABLE_ALIAS}
             {sample_clause}
-            {self._get_person_ids_query(relevant_events_conditions=f"{entity_query} {date_query}")}
```

Comment:
`relevant_events_conditions` is an optimization to reduce the amount of events we need to read from disk – why do we remove it here?

Reply:
My bad. I didn't revert this change.